Setup/Prepare-Release.ps1

```diff
@@ -143,10 +143,10 @@ try {
     Write-Host ----------------------------------------------------------------------
     Generate-Changelog -milestones $milestoneNumbers
 
-    #Write-Host ----------------------------------------------------------------------
-    #Write-Host Download PluginManager
-    #Write-Host ----------------------------------------------------------------------
-    #.\Download-PluginManager.ps1 -ExtractRootPath '..\Plugins\GitExtensions.PluginManager'
+    Write-Host ----------------------------------------------------------------------
+    Write-Host Download PluginManager
+    Write-Host ----------------------------------------------------------------------
+    .\Download-PluginManager.ps1 -ExtractRootPath '..\Plugins\GitExtensions.PluginManager'
```

Comment:
Setup/Prepare-Release.ps1  script is not used during the build pipeline, it is (still) only used for some manual tasks but by far and large it is obsolete